components/ruby/puppet/patches/0001-no-upstream-Solaris-auditing-support.patch

```diff
@@ -0,0 +1,98 @@
+From d68b018f2d30df2824335b579ee3bff4c10cb1e0 Mon Sep 17 00:00:00 2001
```

Comment:
This patch relies on PuppetAudit class, which is provided by ext/*  in upstream repository and likely will need some work to port to OI.